packages/app/src/DBSearchPage.tsx

```diff
@@ -756,8 +759,18 @@ function DBSearchPage() {
     setValue('orderBy', defaultOrderBy);
   }, [inputSource, inputSourceObj, defaultOrderBy]);
 
+  const [isAlertModalOpen, { open: openAlertModal, close: closeAlertModal }] =
+    useDisclosure();
+
   return (
     <Flex direction="column" h="100vh" style={{ overflow: 'hidden' }}>
+      {isAlertModalOpen && (
```

Comment:
can we add a feature flag for alerting components? or using `IS_DEV` flag to expose it in dev environment only

Reply:
done